src/index.test.js

```diff
@@ -0,0 +1,26 @@
+const storageNewElement = require('./index.js');
```

Comment:
- [ ] Hi, nice attempt writing Jest test. Currently, your tests are not passing. when I run `npm test` command, I get the following error:

![image](https://user-images.githubusercontent.com/77106962/159122100-3dd03342-94e8-466f-b349-2065089a1194.png)

The first line has a syntax error. I recommend you use _**Es6 import**_ statements. For example in index.js, you `export` the `storageNewElement` function and then import it using `import { storageNewElement } from './index.js';`. You can do the same for line 2 which has this line `const { default: renderElement } = require('./modules/itemFunctions.js');`